apps/vmq_server/src/vmq_rocksdb_store.erl

```diff
@@ -313,18 +273,18 @@ open_db(Opts, State) ->
 open_db(_Opts, _State0, 0, LastError) ->
     {error, LastError};
 open_db(Opts, State0, RetriesLeft, _) ->
-    case eleveldb:open(State0#state.data_root, State0#state.open_opts) of
+    case rocksdb:open(State0#state.data_root, State0#state.open_opts) of
         {ok, Ref} ->
             {ok, State0#state { ref = Ref }};
         %% Check specifically for lock error, this can be caused if
-        %% a crashed instance takes some time to flush leveldb information
+        %% a crashed instance takes some time to flush rocksdb information
         %% out to disk.  The process is gone, but the NIF resource cleanup
         %% may not have completed.
         {error, {db_open, OpenErr}=Reason} ->
-            case lists:prefix("IO error: lock ", OpenErr) of
+            case lists:prefix("IO error: While lock ", OpenErr) of
```

Comment:
What does the `While` add here? Should it be maybe just `IO error: locked` ?

Reply:
Here we want to match the error message resulting when re-opening a locked RocksDB... apparently the error message looks like this... we can't change that.

Reply:
ha, I'd totally read this as something we wanted to log. Funny how one becomes blind when the mind has been made up :)